src/common/meta/src/key.rs

```diff
@@ -666,7 +659,7 @@ impl TableMetadataManager {
         let operations = self
             .table_metadata_keys(table_id, table_name, table_route_value)?
             .into_iter()
-            .map(|key| TxnOp::Delete(key.into_bytes()))
+            .map(TxnOp::Delete)
```

Comment:
Better to use [batch_delete](https://github.com/GreptimeTeam/greptimedb/blob/09b3c7029b7d7ac8abd1cebb613c9a00ea7c66bb/src/common/meta/src/kv_backend.rs#L65), it has no keys count limit.